### PR TITLE
updated garmin basecamp version 4.4.6 => 4.4.7

### PR DESCRIPTION
**Pull request by MelonSmasher**
This cask gets a 404 at the moment because the version 4.4.6 file is not found. This updates the version to 4.4.7.

### CODE REVIEW

File: Casks/basecamp.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask :v1 => 'basecamp' do
-  version '4.4.6'
+  version '4.4.7'
   sha256 'd114af2d4f68132cee1739eae67a5704bf09612bae032a97535b2befcc44ab40'
```
Comment (vitorgalvao) [inline Casks/basecamp.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by MelonSmasher** [comment]
@vitorgalvao Thanks, sorry about that, this is my first contribution to cask. That sha256 is updated.

**Comment by MelonSmasher** [comment]
I did not realize that this pull request would keep up with my master branch automatically. I created a cask and it looks like it is included in this to?

**Comment by jawshooah** [comment]
@MelonSmasher you should never create pull requests from master to master. Your changes should be isolated in their own topic branch for exactly the reason you alluded to: *any* changes on your branch will be included in the pull request.

**Comment by vitorgalvao** [comment]
@MelonSmasher Yep, what @jawshooah said. Want to try again?

Also, if you plan on submitting that other cask, it needs various fixes, like whitespace between lines and a check to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md).

**Comment by MelonSmasher** [comment]
will do guys thanks.

**Pull request closed without merging**